front/src/components/business/MenuLink/MenuLink.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import Link from 'next/link';
+
+import { Icon } from '@/components/common/Icon';
+import { Typography } from '@/components/common/Typography';
+
+export type MenuLinkProps = {
+  title: string;
+  to: string;
+  badgeNumber?: Number;
+  iconName?: string;
+};
+
+export const MenuLink = ({ title, to, badgeNumber, iconName }: MenuLinkProps) => {
+  return (
+    <Link href={to}>
+      <a className="grid h-[40px] w-[210px] grid-cols-[34px_1fr_20px] content-center rounded-lg border border-black">
+        <div className="flex items-center justify-self-center">
+          {!badgeNumber && iconName && <Icon name={iconName} size="lg" />}
```

Comment:
```suggestion
          {badgeNumber !== undefined && iconName && <Icon name={iconName} size="lg" />}
```

c'est mieux de faire comme ça prsk si `badgeNumber === 0` alors `!badgeNumber` ça sera true.

Reply:
je me suis trompé c'est `badgeNumber === undefined` qu'il faut mettre et ça va marcher

Reply:
Du coup j'ignore la possibilité d'avoir à la fois badgeNumber et iconName

Reply:
La condition pour badge number tu mets `badgeNumber !== undefined`


Et pour icon tu mets  `badgeNumber === undefined && iconName`